documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1080 @@
+<3> Specify a network attachment definition for each additional {virt} network.
+<4> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
+<5> Specify a network attachment definition for each additional {virt} network.
+<6> Required only when `type` is `multus`. Here, `namespace` can either be specified using the namespace property or with a name built as follows: `<network_namespace>/<network_name>`.
+endif::[]
+
+ifdef::vmware[]
+[start=5]
+. Create a `StorageMap` manifest to map source and destination storage:
++
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        id: <source_datastore> <2>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode>
+      source:
+        id: <source_datastore>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the VMware data storage MOR. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
+endif::[]
+
+ifdef::rhv[]
+[start=4]
+. Create a `StorageMap` manifest to map source and destination storage:
++
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        id: <source_datastore> <2>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode>
+      source:
+        id: <source_datastore>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `ReadWriteOnce` and `ReadWriteMany`.
+<2> Specify the {rhv-short} storage domain UUID. For example, `f2737930-b567-451a-9ceb-2887f6207009`.
+endif::[]
+
+ifdef::ova[]
+[start=4]
+. Create a `StorageMap` manifest to map source and destination storage:
++
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: StorageMap
+metadata:
+  name: <storage_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        storageClass: <storage_class>
+        accessMode: <access_mode> <1>
+      source:
+        name:  Dummy storage for source provider <provider_name> <2>
+    - destination:
```

Comment:
here we really need to drop the second mapping since we have only a single "Dummy" storage for an OVA provider